src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -527,12 +527,11 @@ private void computeVisibleMethods(LocalMemberTable lmt) {
         Map<ExecutableElement, List<ExecutableElement>> overriddenByTable = new HashMap<>();
         for (VisibleMemberTable pvmt : parents) {
             // Merge the lineage overrides into local table
```

Comment:
not sure what "lineage" means!

Reply:
I guess the original author used that as a metaphor to denote a branch of inheritance.